salt/metalk8s/kubernetes/etcd/installed.sls

```diff
@@ -12,18 +12,24 @@ include:
 {#- Get the list of existing etcd member. #}
 {%- set etcd_members = pillar.metalk8s.etcd.members %}
 
-{#- Compute the initial state according to the existing list of node. #}
-{%- set state = "existing" if etcd_members else "new" %}
+{%- if pillar.is_bootstrap and not etcd_members %}
```

Comment:
```suggestion
{%- if pillar..get('is_bootstrap') and not etcd_members %}
```

Reply:
Oops! And indeed, there's no sense in always having this key in the pillar.